mule-extensions-api/src/main/java/org/mule/runtime/extension/api/connectivity/OperationTransactionalAction.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.connectivity;
+
+/**
+ * The type of joining actions that operations can take
+ * regarding transactions.
+ *
+ * @since 1.0
+ */
+public enum OperationTransactionalAction
+{
+    /**
+     * There must always be a transaction present for the invocation
+     * <p>
+     * J2EE: Mandatory
```

Comment:
This is how currently defined in TransactionConfig. I'm trying to use the same language.

As for why not changing the naming everywhere, besides the obvious impact there's the fact that we discussed many times that the J2EE naming sucks, so I think it's good to be able to use our own names if we wanted to, while keeping a reference to the standard